app/controllers/api/v1/city_controller.rb

```diff
@@ -1,7 +1,7 @@
 class Api::V1::CityController < ApplicationController
   def index
     city = CityFacade.city_data(params[:city], params[:state_initials])
-    new_city = City.find_or_create_by(city: city.city, state: city.state, summary: city.summary, total_score: city.total_score, categories_hash_array: city.categories_hash_array)
+    new_city = City.find_or_create_by(city: city.city, state: city.state, summary: city.summary, total_score: city.total_score, categories_hash_array: city.categories_hash_array, population: city.population)
     if new_city.save
```

Comment:
Ok, I'm adding most of the city's stats to a favorites object as well, I'll add population to that.